commission_formula/__manifest__.py

```diff
@@ -10,7 +10,7 @@
     "summary": "Commissions computed by formulas",
     "author": "Abstract,Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/commission",
-    "depends": ["commission"],
+    "depends": ["commission", "account"],
```

Comment:
This added dependency doesn't seem correct.

Reply:
Here we used move line https://github.com/OCA/commission/pull/409/commits/f141ae9381f3240cc33a477869006759870d5627#diff-628b4901eac43e5fc9b48afa3b145f4d37d09a8ce33340b551f2297524fbc072R11

Reply:
The link is not correctly shown, but I have checked full code and I don't see such dependency.